string.c

```diff
@@ -11856,18 +11782,131 @@ rb_enc_interned_str_cstr(const char *ptr, rb_encoding *enc)
 }
 
 /*
- *  A String object holds and manipulates an arbitrary sequence of
- *  bytes, typically representing text or binary data. String objects may be
- *  created using String::new or as literals.
+ *  A \String object has an arbitrary sequence of bytes,
+ *  typically representing text or binary data.
+ *  A \String objects may be created using String::new or as literals.
  *
  *  String objects differ from Symbol objects in that Symbol objects are
- *  designed to be used as identifiers, instead of text or data.
+ *  commonly used as identifiers, instead of text or data.
```

Comment:
Please change back from `commonly used` to `designed to be used`.  `commonly used` can imply that such usage is common but not a good idea.